routes/tasks.js

```diff
@@ -126,4 +126,29 @@ router.post('/', authenticate, authorizeOwner, createTask, tasks.addNewTask)
  */
 router.patch('/:id', authenticate, authorizeOwner, updateTask, tasks.updateTask)
 
+/**
+ * @swagger
+ * /tasks/:username
+ *   get:
+ *     summary: Use to get all the tasks of the requested user
+ *     tags:
+ *       - Tasks
+ *     security:
+ *       - bearerAuth: []
+ *     responses:
+ *       200:
+ *         description: returns all tasks of the requested user
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/tasks'
+ *       500:
+ *         description: badImplementation
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/errors/badImplementation'
+ */
+router.get('/:username', authenticate, tasks.getUserTasks)
```

Comment:
This route will be used on the members page. It should NOT require authentication. If a recruiter comes to your page, do you expect them to be logged in, just to be able to view your tasks?

Reply:
I was not aware of where we will integrate this API in frontend. But yeah we don't need authentication. Will remove it.